cmd/podman/run.go

```diff
@@ -30,7 +30,9 @@ var (
 			runCommand.GlobalFlags = MainGlobalOpts
 			return runCmd(&runCommand)
 		},
-		Example: "IMAGE [COMMAND [ARG...]]",
+		Example: `podman run imageID ls -alF /etc
+  podman run --net=host imageID dnf -y install java
+  podman run --volume /var/hostdir:/var/ctrdir --interactive --tty fedora /bin/bash`,
```

Comment:
I think we should swap this to `-t -i` - these are incredibly common options, and people will almost always see the short versions

Reply:
will do.  I figured 98% of the folks would know -i -t, I spelled it out for the 2%.  But it is lengthy.